content/join.hbs

```diff
@@ -59,32 +59,32 @@
 			<div class="columns">
 				<div class="column date">
 					<div class="calendar">
-						<div class="month">November</div>
+						<div class="month">July</div>
 						<div class="day">15</div>
 					</div>
 					<p>Cross-registration Deadline</p>
 				</div>
 				<div class="column date">
 					<div class="calendar">
-						<div class="month">January</div>
-						<div class="day">5</div>
+						<div class="month">July</div>
+						<div class="day">19</div>
 					</div>
 					<p>NSE Package Due</p>
 				</div>
 				<div class="column date">
 					<div class="calendar">
-						<div class="month">January</div>
-						<div class="day">14</div>
+						<div class="month">July</div>
+						<div class="day">5</div>
 					</div>
 					<p>
 						AFROTC Day 1 &mdash; In-processing
 						<br />
-						<small>(0900&ndash;1500)</small>
+						<small>(0730&ndash;1500)</small>
 					</p>
 				</div>
 			</div>
 
-			<h1>Join in <span class="semester-text">Spring 2021</span> or later</h1>
+			<h1>Join in <span class="semester-text">Fall 2021</span> or later</h1>

```

Comment:
I think we talked about this last time but doesn't Spring 2022 make more sense since this link is for future interest?

Reply:
I see what you mean, I will inform Cadre. Thank you